kube/services/jobs/aws-bucket-replicate-job.yaml

```diff
@@ -0,0 +1,92 @@
+# run with 
+# g3k runjob aws-bucket-sync \
+# GDC_BUCKET_NAME a_bucket_name \
+# CHUNK_SIZE chunk_size
+#
+# folow the below link  to setup bucket owner grant access to other aws account 
+# https://docs.aws.amazon.com/AmazonS3/latest/dev/example-walkthroughs-managing-access-example2.html
+#
+# setup bucket policy so that the user test can access to the bucket (GDC bucket)
```

Comment:
Would you give some more context here or in jobs/README.md?
Ex - this job is used to copy objects from GDC to an AWS bucket using a manifest ... something like that?

Reply:
fixed at e9fdab74ac2aa33db7836eb27afa2890695ce597